edgegrid/gtm_client_api.go

```diff
@@ -32,7 +32,7 @@ func (c *GTMClient) DomainStatus(name string) (*ResourceStatus, error) {
 	return status, err
 }
 
-// DomainCreate takes an Akamai GTM domain name and a domain type and creates
+// DomainCreate takes an Akamai GTM domain name and a domainType and creates
 // the domain via the Akamai GTM API.
 func (c *GTMClient) DomainCreate(name string, domainType string) (*DomainResponse, error) {
```

Comment:
The same advise applies for many of the following methods and their documentation.